hasura/migrations/default/1675459146542_create_activities_table/up.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE "public"."activities" (
```

Comment:
@levity i dont need you to fully review this code (unless you want to) but since this is a big new DB architecture, I invite your feedback and thoughts before this is merged as its much easier to incorporate changes earlier rather than later.

Reply:
this looks good to me -- i can see that avoiding more exotic polymorphism patterns would probably be most compatible with Hasura, and also GraphQL generally -- it seems like [the spec is still in flux](https://github.com/graphql/graphql-spec/issues?q=polymorphism+is%3Aopen)